documentation/query-api.md

```diff
@@ -212,3 +214,65 @@ version and a specific detail level:
             }
         }
     }
+
+
+## Simple (plaintext) Endpoints
+
+These endpoints are designed for load balancers that don't support any kind of
+JSON parsing or query parameter use. They return simple string bodies (either
+the state of the service in question or a simple error message) and a status
+code relevant to the status result.
```

Comment:
I think it might be useful/help in clarifying if you say something like if you need non-https for your load balancer, you should use the proxy service with this.